src/engraving/dom/rest.cpp

```diff
@@ -697,6 +697,13 @@ double Rest::rightEdge() const
     return x() + width();
 }
 
+double Rest::centerX() const
+{
+    SymId sym = layoutData()->sym();
+    const auto& bbox = symBbox(sym);
+    return symWidth(sym) / 2 + bbox.bottomLeft().x();
```

Comment:
Calling symBbox and them symWidth is a bit of double work. I'd simplify it to
```suggestion
    return bbox.left() + bbox.width() / 2;
```

Also, `symBbox` does not return `const RectF&` but just `RectF`, so you can just write `RectF bbox = symBbox(sym)`.